project/Base.scala

```diff
@@ -36,6 +36,7 @@ class Base extends Build {
     version := Git.version,
     homepage := Some(url("https://linkerd.io")),
     scalaVersion in GlobalScope := "2.11.7",
+    ivyScala := ivyScala.value.map(_.copy(overrideScalaVersion = true)),
```

Comment:
Oh, we get a ton of warnings from sbt always because, for instance jackson, pulls in alternate scala versions.  I put this in at some point to pin scala versions to ours